README.md

```diff
@@ -2,35 +2,42 @@
+MPS outside of MPS using JVM (Java/Kotlin) or web frameworks/tools.
 
-While all the examples use the same language defined in MPS they aren't meant as one complete example
-but rather as individual parts. The purpose of the examples is to give inspiration of what is possible with modelix 
+None of the examples try to reuse editors defined in MPS.
+If your goal is to reuse MPS editors as is then modelix itself ships with a projector integration to achieve this.
+
+While all the examples use the same language defined in MPS they are not meant as one complete example
+but rather as individual parts. The purpose of the examples is to give inspiration of what is possible with modelix
 and make you think and envision your own use-cases.
 
-## Repository Structure 
+## Repository Structure
+
+The repository is a single gradle project to make building everything at once easy. The examples are organized by the
+technology they use into the various sub-folders:
+
+| ID    | Components/Technologies                               | Description                                                                                                                        | Status | Folder                                                           |
+|-------|-------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|--------|------------------------------------------------------------------|
+| `1`   | MPS + `cloud-plugin`                                  | MPS language definition that is used by all examples. The MPS "structure" is used to generate a Java API consumed by all examples. | ✅      | [mps](mps)                                                       |
+| `2`   | MPS, `api-gen`                                        | Generated Java API from the MPS language.                                                                                          | ✅      | [University.Schedule.api](mps/solutions/University.Schedule.api) |
+| `3a`  | MPS, `json-bulk-access` [EX DASHBOARD FOLDER w/ ktor] | A hand-crafted API that exposes the model contents as a REST API based on MPS as a source for the models                           | 🏗     | [rest-api-json-bulk](rest-api-json-bulk)                         |
+| `3b`  | MPS, `model-server`     [QUARKUS/SPRINGBOOT]?         | A hand-crafted API that exposes the model contents as a REST API based on a `model-server` as a source for the models              | 🏗     | [rest-api-model-server](rest-api-model-server)                   |
+| `4`   | REST, angular                                         | A single page app that realizes a dashboard based on either REST API                                                               | 🏗     | [dashboard](dashboard) single page app                           |
```

Comment:
Is the status icon here correct? Shouldn't this be an X?

Reply:
:heavy_check_mark: